src/components/SimpleSlider.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Slider from 'react-slick';
+import './simple-slider.css';
+
+const SimpleSlider = () => {
```

Comment:
Nice that you got this react-slick slider to work! I struggled a lot with this one? Especially since something in the package made it incredibly big so it wouldn't display properly, did you have the same issue?